drivers/project_scorecard/app/views/project_scorecard/warehouse_reports/scorecards/_report_list.haml

```diff
@@ -1,29 +1,28 @@
-- if @reports.exists?
```

Comment:
It looks like we lost the "No reports on file" text when there are no reports to show.  Will it just show an empty table?  If that's the case, can we bring that back?  Maybe it just moved up the chain, but I'm not seeing it.

Reply:
The `page_entries_info` line does it.

On that note, we had the conversation about moving to Pagy for the other DQ report ticket, but I didn't update this one. I can do it in a follow-up PR.

<img width="1001" alt="Screen Shot 2022-04-11 at 8 37 32 AM" src="https://user-images.githubusercontent.com/5333982/162740742-5b03dee7-8862-4f40-a462-f2b507c6b231.png">

Reply:
That's probably fine.  In other places, we go ahead and hide the table header, it just looks cleaner, but I'm not sure it's important enough to implement here.